dev/com.ibm.ws.classloading/src/com/ibm/ws/classloading/internal/UnifiedClassLoader.java

```diff
@@ -170,6 +171,24 @@ protected Class<?> findClass(String name, DelegatePolicy delegatePolicy, boolean
         throw new ClassNotFoundException(name);
     }
 
+    @Override
+    public InputStream getResourceAsStream(String resName) {
+        if (parent != null) {
+            InputStream fromParent = parent.getResourceAsStream(resName);
+            if (fromParent != null) {
+                return fromParent;
+            }
+        }
+        InputStream fromFollowOn = null;
+        for (ClassLoader cl : followOnClassLoaders) {
+            fromFollowOn = cl.getResourceAsStream(resName);
```

Comment:
My concern is this will end up calling the parent as well for each one of these.